Speech based Emotion Recognition

ELITA04 opened the pull request:
Hi, 
This is with regards to issue #176 

I have added the dataset, notebook for training along with the model. I have also created a simple streamlit application to show the working.

Please review and let me know if any changes are required.

Thank you

ELITA04 committed:
Added notebook and streamlit application for voice based emotion rec

ashishsahu1 commented:
looks good to me, @Niketkumardheeryan what are your thoughts

Niketkumardheeryan commented:
> looks good to me, @Niketkumardheeryan what are your thoughts

@ashishsahu1 audio classification using CNN, it's amazing. But if proper Documentation will be there, it will be great.

Niketkumardheeryan commented:
@ELITA04  please edit readme.md file properly,
you could mention these in the readme.md file-
1. Required libraries for this project
2. Dataset link 
3. Framework or algorithms used in the project .

ELITA04 committed:
Updated readme for Speech Emotion Rec

ELITA04 commented:
Hi @Niketkumardheeryan,
Thank you for your suggestions.

I updated the README.md. I added an overview of the project and also added the libraries used.

Please let me know if anything else is required. Thank you

Niketkumardheeryan commented:
@ashishsahu1 take a look. Now it's looking good.
